authors/settings.py

```diff
@@ -152,4 +152,11 @@
 }
 
 # Activate Django-Heroku; must be at the very bottom
-django_heroku.settings(locals())
\ No newline at end of file
+django_heroku.settings(locals())
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = os.getenv('EMAIL', '')
+EMAIL_HOST_PASSWORD = os.getenv('EMAIL_PASSWORD', '')
+EMAIL_PORT = 587
```

Comment:
You should get all these from environment before setting defaults.

Reply:
fixed issue

Reply:
plus the failing tests